src/styles/StyleUtils.js

```diff
@@ -404,6 +404,16 @@ function parseStyleAsArray(styleParam) {
     return _.isArray(styleParam) ? styleParam : [styleParam];
 }
 
+/**
+ * @param {Number} paddingLeft
+ * @returns {Object}
+ */
+function getPaddingLeft(paddingLeft) {
+    return {
+        paddingLeft,
```

Comment:
@parasharrajat I think we should remove this function from StyleUtils because it isn't generalized enough

Reply:
Then where should I put it? It is a style function. In the future, more sides of padding can be added to it. but they are not needed atm so I didn't add them.

Reply:
> Where should I put it

Hmm, yeah i get you